### ISSUE TITLE
bug: Repeated start works in dictation mode recognition for SDK 1.13.1; does not work for SDK 1.14.x

### ISSUE THREAD

**Issue by nfreear** [microsoft/cognitive-services-speech-sdk-js#299]
Hi,

I think I've found a __bug__ with the SDK version 1.14.x ...

## Expected behaviour

Using version 1.13.1 of the SDK, in dictation mode, I'm able to start continuous speech recognition, stop it or it times out ... then successfully start recognition a second ... third, fourth ... _multiple_ times:

```
https://nfreear.github.io/dictation/?key=_AZURE_SPEECH_KEY__WESTEUROPE_
```

## Actual behaviour

Using version 1.14.0 / 1.14.1 of the SDK in dictation mode, I can start continuous speech recognition once and it recognizes, but when I start it a second time, no speech is recognized:

```
https://nfreear.github.io/dictation/bug.html?key=_AZURE_SPEECH_KEY__WESTEUROPE_
```

### Notes

 * The above demo URLs use the `westeurope` region, and an Azure cognitive services speech resource subscription key (_please create your own to test_);
 * Chrome 87.0.4280.88 + other browsers;
 * (Mac OS 10.13.6)
 *  Endpoint URL :~ `wss://westeurope.stt.speech.microsoft.com/speech/recognition/dictation/cognitiveservices/v1?initialSilenceTimeoutMs=6000&endSilenceTimeoutMs=3000&`
 * Demo code :~ https://github.com/nfreear/dictation;
 * Demo URL :~ https://nfreear.github.io/dictation/

I found this bug while working on the _Microsoft AI for Accessibility_-funded, "_ADMINS_" research project.

Yours,

Nick

---
### Expected screenshot
![speech-sdk-dictation-v1 13 0-no-bug-expected_2](https://user-images.githubusercontent.com/131235/103640285-a7b71d00-4f47-11eb-836f-733fc3de4e28.jpg)

### Actual screenshot
![speech-sdk-dictation-v1 14 0-with-bug-actual_1](https://user-images.githubusercontent.com/131235/103640313-b1d91b80-4f47-11eb-924a-c9fd88432914.jpg)

[bug]

**Comment by nfreear** [comment]
## Correction

Correction :~ the demo URLs use the `westeurope` region (_NOT_ the `uksouth` region!)

---

SDK properties ...

![speech-sdk-dictation-v1 13 0-properties_3](https://user-images.githubusercontent.com/131235/103641496-c0c0cd80-4f49-11eb-93ff-a307b2d956b8.jpg)


Nick

**Comment by glharper** [comment]
@nfreear This appears to be the same issue as [this one](https://github.com/Azure-Samples/cognitive-services-speech-sdk/issues/881). This issue was fixed in [this PR](https://github.com/microsoft/cognitive-services-speech-sdk-js/pull/283), and the fix will be included in 1.15, due within the next couple of weeks.

**Comment by nfreear** [comment]
Hi Glenn,

Thanks for responding and linking to the issue in the `Azure-Samples` org. Good to hear that a fix is in the pipeline.

Best wishes,

Nick
